run-bungeecord.sh

```diff
@@ -89,6 +91,27 @@ do
 done
 fi
 
+# Download rcon plugin
+if [ "${ENABLE_RCON^^}" = "TRUE" ] && [[ ! -e $BUNGEE_HOME/plugins/${RCON_JAR_URL##*/} ]]; then
+  echo "Downloading rcon plugin"
+  if ! curl -sSL -o /tmp/${RCON_JAR_URL##*/} $RCON_JAR_URL; then
+    echo "ERROR: failed to download from $RCON_JAR_URL to /tmp/${RCON_JAR_URL##*/}"
+    exit 2
+  fi
+
+  mkdir -p $BUNGEE_HOME/plugins/bungee-rcon
```

Comment:
You could move this before the curl on L97 and then `-o` directly into there. Worst case if the curl fails, it leaves behind an empty directory.